### ISSUE TITLE
Vote-4-DaJuice

### ISSUE THREAD

**Issue by dajuice21** [Degenerate-Ape-Academy/the-daoo#22]
#### You can view my apes at this address: 5QtqzXJDQbbVZrrWJhWcEFMZpCkcEM4WqBJLbGiPTQy5
---

# You should vote for me because:

**TLDR: I'm not a whale but have a bunch of Apes. I'm a CPA IRL and want to make sure we all make a shit ton of money from our Apes.**

I am the Golden King Ape.  It is only fitting that I serve on the inaugural Council!
![eb66c346-31fd-4884-bf28-08ee53cd1ea0](https://user-images.githubusercontent.com/91101652/134107021-889ee9d1-40b6-456c-9717-4b5866acb977.png)

Seriously though, I have been involved in the project since mint, own 20+ Apes, and am active in the Discord and on Twitter promoting our Apes.  I am committed to seeing this project succeed.  

IRL I own a CPA firm and I will bring my decade+ of experience working with small and medium business owners with me to the Council to help us monetize the shit out of our Apes!

# If I'm elected, I will:
Continue to put my focus on growing the DAA community and increase the value of our Apes.  

I will work with the other Council Members to listen to all of the issues and proposals put forth by the Academy Members and will work with the development team to make sure the best ideas are implemented to make our community the premier in the crypto universe.

I am in favor of an amicable resolution to the Exiled Ape issue, which may or may not include full acceptance into the Academy depending on the different options presented by the development team.  My main objective is to make sure the 10k Apes are not devalued.

Stay tuned, more updates coming in 2 hours!
